src/connector/Cargo.toml

```diff
@@ -71,7 +71,7 @@ jni = { version = "0.21.1", features = ["invocation"] }
 jsonbb = { workspace = true }
 jsonwebtoken = "9.2.0"
 maplit = "1.0.2"
-moka = { version = "0.12.0", features = ["future"] }
+moka = { version = "0.12.8", features = ["future"] }
```

Comment:
Can we still use `0.12.0` in this PR, so that there's no changes in `Cargo.lock`and bumping of `quanta` can be reviewed in a separate PR?

Reply:
I am afraid not. We are using `and_try_compute_with`, which is unavailable in v0.12.0.

Reply:
Would you try `0.12.3`?

Reply:
I think it works

Reply:
You may want to checkout the `Cargo.lock` on the main branch to downgrade the version of `quanta`.

Reply:
![image](https://github.com/user-attachments/assets/72bcce9c-f361-49f6-96da-4df82e92f72c)

moka 0.12.3 still needs a higher version of `quanta`. I am afraid we have to bump quanta first.

Reply:
You're right. 😢 In its repo the version `0.12.3` released before bumping dependency on `quanta`, but on crates.io it's the other way around.